manager/views.py

```diff
@@ -319,7 +323,9 @@ def schedule(request, event_slug):
         return render(request, 'activities/schedule.html',
                       update_event_info(event_slug, event=event, render_dict={'schedule': schedule}))
     messages.warning(
-        _("You don't have confirmed activities, please confirm the activitiees and then confirm the schedule"))
+        request,
+        _("You don't have confirmed activities, please confirm the activitiees and then confirm the schedule")
```

Comment:
You don't have any confirmed activities. Please confirm the activities first and then confirm the schedule